### PR TITLE
5541/5558 - Fix for Lookup/Timepicker icon not rendering properly in Safari. Fix for Spinbox box shadow not rendering properly

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixes a bug for both lookup and timepicker for short field and its icons not rendering properly.
It will also fix the box-shadow in spinbox-sm.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5541
Closes https://github.com/infor-design/enterprise/issues/5558

**Steps necessary to review your pull request (required)**:
**#5541**
**1st Issue**
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/validation/example-short-fields.html using all your browsers
- Click on spinbox-sm

**2nd Issue**
- Go to http://localhost:4000/components/validation/example-short-fields.html & http://localhost:4000/components/validation/example-multiple-errors.html using Safari
- Click on lookup icon on example multiple errors & lookup and timepicker icons on example short fields

**3rd Issue**
- Go to http://localhost:4000/components/validation/example-short-fields.html using Firefox
- Click on Lookup & Datepicker button

**#5558**
- Go to http://localhost:4000/components/timepicker/example-index.html?locale=zh-Hans (CN/Hant/TW) using Safari
- Click on Timepicker icon

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by tmcconechy** [comment]
This looks good. FIxed all the main issues but try tabbing into the trigger buttons in all the browsers and they all have some degree of misalignment on the icon / focus state.

Screen Shot 2021-09-02 at 10 10 37 AM

Should have the icon and focus sate centered...(QA might pick this up)

**Review by tmcconechy — DISMISSED** [review]
nice 👍🏻  looks great now

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # What's New with Enterprise
 
+## v4.56.0 Fixes
+
```
Comment (ericangeles) [inline docs/CHANGELOG.md:4]: @tmcconechy should also put this on 4.55 Fixes

**Review by ericangeles — APPROVED** [review]
But overall looks good to me! 🍌

File: docs/CHANGELOG.md (hunk 1 above)
Comment (tmcconechy) [reply docs/CHANGELOG.md:4]: Yes but i can do it when i release

**Review by tmcconechy — APPROVED** [review]

**Comment by tmcconechy** [comment]
Actually did it in the browser.

**Pull request merged**